src/screens/TopicPageScreen/index.js

```diff
@@ -107,6 +106,12 @@ const TopicPageScreen = (props) => {
     extrapolate: 'clamp'
   });
 
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces an `opacityImage` variable that is interpolated based on the `scrollY` value. This seems to be used for some kind of animation effect, but without seeing how it's used in the rest of the code, it's hard to say if there are any issues with this change. However, from what I can see, the logic seems sound.

```diff
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->